README.md

```diff
@@ -1,2 +1,92 @@
-# pools
-Lite &amp; fast micro PHP pools library that is **easy to use**.
+# Utopia Pools
+
+[![Build Status](https://travis-ci.org/utopia-php/abuse.svg?branch=master)](https://travis-ci.com/utopia-php/pools)
+![Total Downloads](https://img.shields.io/packagist/dt/utopia-php/pools.svg)
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord)](https://appwrite.io/discord)
+
+Utopia pools library is simple and lite library for managing long living connection pools. This library is aiming to be as simple and easy to learn and use. This library is maintained by the [Appwrite team](https://appwrite.io).
+
+Although this library is part of the [Utopia Framework](https://github.com/utopia-php/framework) project it is dependency free, and can be used as standalone with any other PHP project or framework.
+
+## Concepts
+
+* **Pool** - A list of long living connections. You can pop connections out and use them and push them back to the pool for reuse.
+* **Connection** - An object that holds a long living database or other external connection in a form of a resource. PDO object or a Redis client are examples of resources that can be used inside a connection.
+* **Group** - A group of multiple pools.
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require utopia-php/pools
```

Comment:
Should we also have an example showing how you can install a specific version ? 

`composer require utopia-php/pools:0.0.1`

Reply:
I would skip it just for the added maintenance in each release.

Reply:
okay 👌